suffix_update.py

```diff
@@ -0,0 +1,132 @@
+import sys
```

Comment:
This file should include a top-level docstring explaining what it is, what it does and how to use it.